docs/getting_started.rst

```diff
@@ -0,0 +1,45 @@
+Getting Started
+===============
+
+.. note::
+
+    There's no single true path to install dunst. Just execute the following commands and in case of an error, lookup the error message in the linked troubleshooting section.
+
+Ensure a DBus session
+---------------------
+
+DBus has to be running in your session::
+
+    dbus-send --session --dest=org.freedesktop.DBus / org.freedesktop.DBus.Peer.Ping && echo "DBus works!"
+
+It has to print out ``DBus works!``, otherwise please consult :ref:`trouble-dbus-unsuccessful`.
+
+Install dunst
+-------------
+
+Mostly, just install the ``dunst`` package with your package manager. For more detailed information, please consider `the installation page <./installation.html>`_.
+
+Alternatively you can `compile dunst from source <./compiling.html>`_.
+
+Run dunst
+---------
+
+Open your terminal and start the dunst **server** for validation::
+
+    dunst -startup_notification
+
+A small window should pop up saying ``dunst is up and running`` for at least a few seconds.
+
+Relogin to your session
+-----------------------
+
+Now, after you've installed and made sure that dunst is able to start, please log out and log in again. Dunst ist now running and receives notifications.
```

Comment:
> Now, after you've installed and made sure that dunst is able to start, please log out and log in again.

Why? Dunst is running, there is nothing to reset here.

Reply:
My point was, that there are between the installation of the package `dunst` and a working dunst instance many factors.

And one common factor, is that there's either no DBus at all. (wrong initialised session, etc.) or there's another notification daemon already running.

And at this point, we assured that both problematic conditions are solved (I see, I haven't added some docs how to handle a competing notification daemon). So we can give DBus the responsibility to start the service. Because if DBus can't or doesn't start our daemon, we do not have good logs or commands to debug it.

It's to assure that it can work. That's it.

----

But yeah actually that's a one reason, why I wanted to have a review. If that's counterintuitive for you. I'd definitely like to improve it. Getting dunst to run is the hardest part and we only can solve it by writing good docs.

I also appreciate radically different approaches!